system-parachains/people/people-kusama/src/lib.rs

```diff
@@ -306,6 +310,7 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 	type CheckAssociatedRelayNumber = RelayNumberMonotonicallyIncreases;
 	type ConsensusHook = ConsensusHook;
 	type WeightInfo = weights::cumulus_pallet_parachain_system::WeightInfo<Runtime>;
+	type SelectCore = cumulus_pallet_parachain_system::DefaultCoreSelector<Runtime>;
```

Comment:
I just checked and we actually enabled async backing on this system chain. We need to use `LookaheadCoreSelector` strategy. Keep in mind that this stuff doesn't have any effect now, but we should have it done properly to avoid any kind of issues when we enable UMP signals in the relay runtime(right after this upgrade). CC @alindima